packages/node/src/node.ts

```diff
@@ -49,6 +51,7 @@ export class Node {
   // initialized in the `asynchronouslySetupUsingRemoteServices` function
   private signer!: HDNode;
   protected requestHandler!: RequestHandler;
+  public router: NodeRouter = {} as NodeRouter;
```

Comment:
Shouldn't this be private?

Reply:
Once we reimplement `Node.call` to use the JSONRPC `router.dispatch()`, we can make it private/protected.